packages/venia-ui/lib/components/ButtonGroup/button.css

```diff
@@ -1,37 +1,37 @@
 .root {
     composes: root from '../clickable.css';
     background: none white;
-    border: 1px solid rgb(var(--venia-grey-dark));
-    color: rgb(var(--venia-text));
+    border: 1px solid rgb(var(--venia-global-color-gray-dark));
+    color: rgb(var(--venia-global-color-text));
     font-size: 0.75rem;
```

Comment:
Should we also be replacing all uses of `font-size: X` with `--venia-global-fontSize-X`?

It seems like the tokens we changed are the old `--venia-FOO` tokens, which is a great start but there are a few tokens that we could also globally apply such as colors font size, weight and line height.

@jimbo what do you think?

Reply:
Yes, we need to do that, but no, not as part of this story.